src/mlpack/methods/softmax_regression/softmax_regression_impl.hpp

```diff
@@ -121,14 +121,22 @@ void SoftmaxRegression<OptimizerType>::Predict(const arma::mat& testData,
 }
 
 template<template<typename> class OptimizerType>
+void SoftmaxRegression<OptimizerType>::Classify(const arma::mat& dataset,
+                                                arma::Row<size_t>& labels)
+    const
+{
+  Predict(dataset, labels);
+}
```

Comment:
We can change `Predict()` to wrap `Classify()` instead; that will make the transition easier to remove `Predict()` for 3.0.0.